model/note/import.go

```diff
@@ -67,12 +68,66 @@ func ImportFile(inst *instance.Instance, newdoc, olddoc *vfs.FileDoc, body io.Re
 	return nil
 }
 
-func importReader(inst *instance.Instance, doc *vfs.FileDoc, reader io.Reader, schema *model.Schema) (*model.Node, error) {
+func ImportImages(inst *instance.Instance, olddoc *vfs.FileDoc) error {
+	inst.Logger().WithNamespace("notes").
+		Infof("importing images from note: %s", olddoc.ID())
+	schemaSpecs := DefaultSchemaSpecs()
+	specs := model.SchemaSpecFromJSON(schemaSpecs)
+	schema, err := model.NewSchema(&specs)
+	if err != nil {
+		return fmt.Errorf("failed to read note schema: %w", err)
+	}
+
+	fs := inst.VFS()
+	file, err := fs.OpenFile(olddoc)
+	if err != nil {
+		return fmt.Errorf("failed to open file for note images import: %w", err)
+	}
+
+	content, images, err := importReader(inst, olddoc, file, schema)
+	cleanImages(inst, images) // XXX: remove images found in the archive but not in the markdown
+	if cerr := file.Close(); cerr != nil {
+		return fmt.Errorf("error while closing note file: %w", cerr)
+	}
+	if content == nil || !hasImages(images) {
+		inst.Logger().WithNamespace("notes").
+			Infof("No images to import")
+		return nil
+	}
+
+	md := markdownSerializer(images).Serialize(content)
+	body := []byte(md)
+	body, err = buildArchive(inst, []byte(md), images)
```

Comment:
you can compact the two lines in one `body, err := buildArchive(inst, []byte(md), images)`